src/assets/scss/modules/_forms.scss

```
@@ -512,35 +519,113 @@
     input,
     label {
       font-family: settings.$dp-font-family-proximanova;
+      font-size: 14px;
+    }
+
+    input {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
+
+    .dp-accept-policies,
+    .dp-accept-promotions {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
+
+    .error .wrapper-errors.dp-error-form p,
+    .error .wrapper-errors.dp-error-form span,
+    .error .wrapper-errors p,
+    .error .wrapper-errors span {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
+
+    .primary-green {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
     }
   }
   /* login form */
 
   .login-form {
     margin-top: var.$dp-spaces--lv6;
 
+    label {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 14px;
+    }
+
+    input {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
+
     .legend {
       font-size: var.$dp-sizing--lvl2;
       float: right;
       color: colors.$dp-color-silver;
       font-style: italic;
       font-weight: settings.$dp-font-weight-light;
     }
+
+    .error .wrapper-errors p,
+    .error .wrapper-errors span,
+    .error .wrapper-errors button {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
+
+    .primary-green {
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
+      font-size: 16px;
+    }
   }
 
   .forgot-link {
     display: block;
     margin: var.$dp-spaces--lv4 var.$dp-spaces--lv0;
-    font-size: var.$dp-sizing--lvl2;
     font-style: italic;
     font-family: settings.$dp-font-family-proximanova;
     position: relative;
+    /* Agregar variables de fonts-size cuando se modifiquen a rem */
+    font-size: 16px;
```

Algo que me suena del todo es que se especifique directamente este tamaño directamente en el atributo, prefiero usar una variable que en el futuro me permita cambiar el tamaño en un solo sitio.
```suggestion
    font-size: var.$font-size-base;
```

---

👍 Hablamos con @Cromeror de este tema y coincidimos en varias ideas, agrego una variable con el tamaño base en el mismo archivo para poder modificarlo en el futuro con los cambios en el proceso de registración que se viene.

---

Con esto buscamos mejorar la semántica para nosotros los **_dev_** durante el mantenimiento de la librería y nos da un punto de extensión que puede ser valioso en el mediano plazo.